src/styles/upload.css

```diff
@@ -147,6 +147,8 @@ main {
   width: 100%;
   height: 100%;
   object-fit: contain;
+  position: relative;
```

Comment:
If this new relative overrides some other relative, should some other css class remove `position:relative`?